CA_DataUploaderLib/CommandHandler.cs

```diff
@@ -55,6 +55,7 @@ public ChannelReader<DataVector> GetReceivedVectorsReader()
         public bool IsRunning => !_exitCts.IsCancellationRequested;
         public CancellationToken StopToken => _exitCts.Token;
         public Task RunningTask => _runningTaskTcs.Task;
+        public List<LoopControlDecision> Decisions => _decisions;
```

Comment:
Consider either returning a copy or something that can't be changed

Reply:
changed to be a IReadOnlyList